.github/workflows/build_conda_linux.yml

```diff
@@ -94,6 +94,10 @@ jobs:
           repository: ${{ inputs.test-infra-repository }}
           ref: ${{ inputs.test-infra-ref }}
           path: test-infra
+      - name: Setup SSH
+        uses: ./test-infra/.github/actions/setup-ssh
+        with:
+          github-secret: ${{ github.token }}
```

Comment:
Please add `teardown-linux` action at the end of the workflow (that runs `always`) to wait for the SSH session to end